modules/gitops-release-notes-1-4-1.adoc

```diff
@@ -18,4 +18,4 @@ The following issue has been resolved in the current release:
 ** `argoproj.io_appprojects.yaml`
 ** `argoproj.io_argocds.yaml`
 
-Before this update, if you created an `AppProject` resource using the `kubectl create`, it failed to synchronize. This update restores the missing description fields in `CustomResourceDefinitions: ArgoCD, AppProject, Application` that caused `kubectl create` to fail. link:https://issues.redhat.com/browse/GITOPS-1721[GITOPS-1721]
\ No newline at end of file
+Before this update, if you created an `AppProject` resource using the `oc create`, it failed to synchronize. This update restores the missing description fields in `CustomResourceDefinitions: ArgoCD, AppProject, Application` that caused `oc create` to fail. link:https://issues.redhat.com/browse/GITOPS-1721[GITOPS-1721]
```

Comment:
```suggestion
Before this update, when you created an `AppProject` resource using the `oc create` command, the resource failed to synchronize due to the missing description fields. This update restores the missing description fields in the preceding CRDs. link:https://issues.redhat.com/browse/GITOPS-1721[GITOPS-1721]
```
**Question:** What is `CustomResourceDefinitions: ArgoCD, AppProject, Application`? An API object?

Please rewrite for more clarity. Refer to the guidelines [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting).

- This issues needs to be indented so that it is inline with the bullet item. Suggest adding `+` at L20.